plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestPartitionFields.java

```diff
@@ -47,6 +48,21 @@ public void testParse()
         assertParse("truncate(comment, 13)", partitionSpec(builder -> builder.truncate("comment", 13)));
         assertParse("truncate(order_key, 88)", partitionSpec(builder -> builder.truncate("order_key", 88)));
         assertParse("void(order_key)", partitionSpec(builder -> builder.alwaysNull("order_key")));
+        assertParse("YEAR(ts)", partitionSpec(builder -> builder.year("ts")), "year(ts)");
+        assertParse("MONtH(ts)", partitionSpec(builder -> builder.month("ts")), "month(ts)");
+        assertParse("DaY(ts)", partitionSpec(builder -> builder.day("ts")), "day(ts)");
+        assertParse("HoUR(ts)", partitionSpec(builder -> builder.hour("ts")), "hour(ts)");
+        assertParse("BuCKET(order_key, 42)", partitionSpec(builder -> builder.bucket("order_key", 42)), "bucket(order_key, 42)");
+        assertParse("TRuncate(comment, 13)", partitionSpec(builder -> builder.truncate("comment", 13)), "truncate(comment, 13)");
+        assertParse("TRUNCATE(order_key, 88)", partitionSpec(builder -> builder.truncate("order_key", 88)), "truncate(order_key, 88)");
+        assertParse("VOId(order_key)", partitionSpec(builder -> builder.alwaysNull("order_key")), "void(order_key)");
+        assertParse(" comment ", partitionSpec(builder -> builder.identity("comment")), "comment");
+        assertParse("day( ts )", partitionSpec(builder -> builder.day("ts")), "day(ts)");
+        assertParse("truncate( comment ,        13 )", partitionSpec(builder -> builder.truncate("comment", 13)), "truncate(comment, 13)");
+        assertParse("void( order_key )", partitionSpec(builder -> builder.alwaysNull("order_key")), "void(order_key)");
+        assertParse(" day( ts ) ", partitionSpec(builder -> builder.day("ts")), "day(ts)");
+        assertParse("  truncate( comment ,13 )  ", partitionSpec(builder -> builder.truncate("comment", 13)), "truncate(comment, 13)");
+        assertParse("   void( order_key )   ", partitionSpec(builder -> builder.alwaysNull("order_key")), "void(order_key)");
```

Comment:
Why would we want to add support for surrounding spaces? looks like redundant flexibility to me.

Reply:
I will remove it. The reasoning was to make it behave more or less as SQL but it is definitely not a necessity.